docs/source/features_conceptual/certainty.rst

```diff
@@ -0,0 +1,35 @@
+.. _TEMPLATE:
```

Comment:
This file is incomplete; it's still the template! I'm assuming this is still in progress?